src/js/index.js

```diff
@@ -0,0 +1,129 @@
+import '../styles/style.css';
+import icon from '../../img/icon.svg';
+import * as add from './add';
+import * as store from './store';
```

Comment:
Please consider using named imports as rather than * as it indicates clearly what you imported